core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -1305,16 +1301,13 @@ public void registerNewWorkerConfListener(BiConsumer<Address, List<ConfigPropert
    * Class that contains metrics related to BlockMaster.
    */
   public static final class Metrics {
-    private static final Counter TOTAL_BLOCKS =
-        MetricsSystem.counter(MetricKey.MASTER_TOTAL_BLOCKS.getName());
-
     /**
      * Registers metric gauges.
      *
      * @param master the block master handle
      */
     @VisibleForTesting
-    public static void registerGauges(final BlockMaster master) {
+    public static void registerGauges(final DefaultBlockMaster master) {
```

Comment:
This is an inner class, we might as well use the specific implementation of the block master and be able to instrument its internals.